src/Network/Network/help/Add-AzLoadBalancerRuleConfig.md

```diff
@@ -79,6 +104,36 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -BackendAddressPools
+Specifies the backend address pools to associate with a load balancer rule configuration. Currently being used for Gateway Load Balancer.
+
+```yaml
+Type: Microsoft.Azure.Commands.Network.Models.PSBackendAddressPool[]
+Parameter Sets: SetByBackendAddressPools
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -BackendAddressPoolIds
+Specifies the IDs of a **BackendAddressPool** object to associate with a load balancer rule configuration. Currently being used for Gateway Load Balancer.
```

Comment:
Is this okay?

Reply:
Specifies the IDs of a **BackendAddressPool** object to associate with a load balancer rule configuration. This can currently only be used when SKU is Gateway.